appveyor.yml

```diff
@@ -25,3 +25,10 @@ test_script:
   - node --version
   - npm --version
   - cmd: npm test
+
+deploy:
+  on:
+    branch: master
```

Comment:
This stuff ^ doesn't appear to work, and their documentation doesn't really elaborate on how it'd be possible to do this (apart from that tag & branch selectors seem to be mutually exclusive)

Reply:
I contacted Appveyor about this in particular at some point. This is what we do in Etcher (adapted to your use case):

```
deploy_script:
  - if %APPVEYOR_REPO_BRANCH%==master (npm run prebuild -- -u %GITHUB_TOKEN%)
```

Reply:
Ah, I see – thanks!